proposals/description/swap_dpi_to_dai.ts

```diff
@@ -0,0 +1,62 @@
+import { ProposalDescription } from '@custom-types/types';
+
+const swap_dpi_to_dai: ProposalDescription = {
+  title: 'Swap DPI to DAI',
+  commands: [
+    {
+      target: 'dpi',
+      values: '0',
+      method: 'transfer(address,uint256)',
+      arguments: ['{dpiToDaiSwapper}', '37888449801955370645659'],
+      description: 'Transfer DPI from DAO timelock to the LBP pool'
+    },
+    {
+      target: 'compoundDaiPCVDeposit',
+      values: '0',
+      method: 'withdraw(address,uint256)',
+      arguments: ['{dpiToDaiSwapper}', '187947000000000000000000'],
+      description: 'Withdraw Use the PCVGuardian to transfer DAI from the CompoundPCVDeposit to the LBP pool'
+    },
+    {
+      target: 'collateralizationOracle',
+      values: '0',
+      method: 'addDeposit(address)',
+      arguments: ['{dpiToDaiLensDai}'],
+      description: 'Add DAI swapper lens to the CR oracle'
+    },
+    {
+      target: 'collateralizationOracle',
+      values: '0',
+      method: 'addDeposit(address)',
+      arguments: ['{dpiToDaiLensDpi}'],
+      description: 'Add DPI swapper lens to the CR oracle'
+    },
```

Comment:
It would be good to prep the actions to withdraw everything and clean up the CR oracle after

Reply:
Yep, good point, will do